example_test.go

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
No. It should actually be `2016-2017`, but I'm punting that whole thing until I can write a tool to handle it properly. 

I'll forward you the whole email chain with legal.